src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  let cashBack = 0;
+
+  if (days > 0 && days < 3) {
+    return rent * days;
+  }
+
+  if (days >= 3 && days < 7) {
+    cashBack = 20;
```

Comment:
if you create discount variables at the very beginning you won't have to set different value for cashback here, just use the existing variable